lib/modules/manager/helmfile/__fixtures__/helmfile.lock

```diff
@@ -0,0 +1,7 @@
+version: 0.149.0
```

Comment:
Please also add test cases in which the lock file is corrupted e.g. not valid yaml  and where it is containing multiple dependencies.

Reply:
Found a few things that slipped through when trying to add the test cases you suggested. Code coverage for `lib/modules/manager/helmfile/artifacts.ts` is better now, but I'm unsure if the tests are testing the behaviour you describe. Do you have any pointers on how to test for the things you suggest?